client/src/utils/gen-util.js

```diff
@@ -4,3 +4,15 @@ export const keyedString = (str, keyValues) =>
     str
   );
 export const capitalizedString = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+
+/**
+ * for use with _orderBy lodash function
+ * _orderBy sorts in a case sensitive way like: [1, Z, a, b, c]
+ * so check if field is not numeric, lowercase it. toString for safety.
+ * @param {{}} item:
+ * @param {string} orderBy: field to order by
+ * @returns []
+ */
+export const orderByCaseInsensitive = (item, orderBy) => {
```

Comment:
This function looks like it's conditionally lowercasing property values rather than ordering things, trying to come up with an accurate name for this function is a bit difficult which makes me feel like we're a bit off-pattern here